src/net/larse/lcms/algorithms/VCT.java

```diff
@@ -1337,8 +1085,7 @@ private void setDisturbance(int startYear, int endYear) {
   }
 
   /**
-   * extract segment data in ud B5 over given time span for linear fit
-   *
+   * Extract segment data in ud B5 over given time span for linear fit
    * This is based on the logic in calculateLinearChangeRate.
    *
    * @param start start of segment inclusive
```

Comment:
I think this function is allocating memory per segment.  Yang says he has the fix.